CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ This file is used to list changes made in each version of the jenkins cookbook.
 
 ## Unreleased
 
+- Remove runit dependency
+- Use systemd units instead of runit services
```

Comment:
As we are not handling the conversion on a running system I consider this a breaking change hence the major release label change. If we were to stop the existing `runit` service prior to bringing up the new service and provide a clean migration path we could release this as a minor release.

Reply:
i.e. I think you should move this into the section below so folks know it's a breaking change.